src/main/java/solvd/laba/ermakovich/hu/repository/elastic/criteria/field/doctor/DepartmentFiled.java

```diff
@@ -0,0 +1,23 @@
+package solvd.laba.ermakovich.hu.repository.elastic.criteria.field.doctor;
+
+import org.springframework.data.elasticsearch.core.query.Criteria;
+import org.springframework.stereotype.Component;
+import solvd.laba.ermakovich.hu.domain.DoctorSearchCriteria;
+import solvd.laba.ermakovich.hu.repository.elastic.criteria.field.Field;
+
+/**
+ * @author Ermakovich Kseniya
+ */
+@Component
+public class DepartmentFiled implements Field {
```

Comment:
@kermakovich `DepartmentFiled`? why not a simple `Department`?

Reply:
agree, it will be better in this case